docs/src/documentation/03-components/03-information/badge/03-accessibility.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/badge/accessibility/
+---
+
+## Accessibility
+
+The Badge component has been designed with accessibility in mind.
+
+### Screen reader support
+
+The Badge component supports screen readers through the `ariaLabel` prop. This prop is especially important when:
+
+- The badge contains only an icon
```

Comment:
With the current `Badge` implementation (`div` in `BadgePrimitive` doesn't have any role), if the icon is the only content the `aria-label` won't be read if I understand correctly. This is also in the violations:

<img width="485" alt="violation1" src="https://github.com/user-attachments/assets/ca597dd1-89c5-4f0b-9f30-8574e54a2033" />

And even if I add some content (e.g. your example), it still says that _aria-label attribute is not well supported on a div with no valid role attribute._ 😬 

<img width="604" alt="violation2" src="https://github.com/user-attachments/assets/b2fa90d6-50c0-4a6e-90ad-e4fe1c65a8d3" />

Reply:
I guess adding a `role="status"` can help here, nice catch!